docs/en/operation/Metrics-System.md

```diff
@@ -91,35 +89,33 @@ sink.csv.directory=/tmp/alluxio-metrics
 
 If Alluxio is deployed in a cluster, this file needs to be distributed to all the nodes.
 
-Then, start Alluxio, CSV files containing metrics will be found in the `sink.csv.directory`. The
-filename will correspond with the metric name.
+Then, start Alluxio, CSV files containing metrics will be found in the `sink.csv.directory`. 
```

Comment:
Instead of `Then, start Alluxio, ....` I would write `After starting Alluxio, the ...`